### DISCUSSION TITLE
Image Browser updated fork

### DISCUSSION THREAD

**Discussion by mark-wd** [vladmandic/sdnext#982] [General]
Image Browser is one of the pre-packed extensions in Automatic (and it reinstalls even if manually removed upon update), hence this discussion.

The original author of the extension seems to be in hiatus, and development has continued under this fork: [Github](https://github.com/AlUlkesh/stable-diffusion-webui-images-browser)

Given that it is producing erros in the startup log as described in #980, should Automatic either move to the new fork, or exclude the extension?

**Comment by vladmandic** [comment]
It's already new fork, always been.

**Comment by mark-wd** [comment]
LOL case closed

On the same general subject... should extensions like these be removable? This was a diferentiation this fork used to have, but apart from things like Lycoris and ControlNet, maybe it's not as relevant nowadays to include a lot of these extensions built-in?

**Reply by vladmandic** [reply]
Which extensions should be built in is a longer discussion. Default is what I used regularly before this repo even became popular.
